### PR TITLE
Make infile buffer size configurable.

### PR DESCRIPTION
**Pull request by io-german**

### CODE REVIEW

**Comment by ph1lm** [comment]
lgtm

**Comment by io-german** [comment]
@amir20 @corcorb Could you please merge this PR if you don't have any new comments. 
Thanks.

**Comment by io-german** [comment]
@amir20 @corcorb ping

**Comment by amir20** [comment]
I think you are missing some test cases. I know it's simple but we should make sure the constructor works with the new parameters. 

@corcorb should merged because I am no longer the primary owner.

**Comment by amir20** [comment]
Make sure you rebase.

**Comment by corcorb** [comment]
Yeah, I can merge this once it is rebased.

**Comment by io-german** [comment]
I have added some tests and rebased the branch.

File: src/main/java/com/opower/persistence/jpile/infile/InfileDataBuffer.java [hunk 1]
```diff
@@ -113,6 +116,10 @@ public InfileDataBuffer(Charset charset, int infileBufferSize, int rowBufferSize
         this.infileBuffer = ByteBuffer.allocate(infileBufferSize);
     }
 
+    public InfileDataBuffer(int infileBufferSize, int rowBufferSize) {
```
Comment (corcorb) [inline src/main/java/com/opower/persistence/jpile/infile/InfileDataBuffer.java]: OK, I know I said I'd merge this, but one final comment (take it or leave it): Do you think it's worth switching this to use a builder and deprecating all the constructors? We have a bunch of constructors that take various arguments, and there are two integer arguments that could be transposed…

Comment (io-german) [inline src/main/java/com/opower/persistence/jpile/infile/InfileDataBuffer.java]: I agree. Builder will look much cleaner than this bunch of constructors. Let's not merge this PR at the moment. I'll update it ASAP.

Comment (io-german) [inline src/main/java/com/opower/persistence/jpile/infile/InfileDataBuffer.java]: Done.

**Pull request merged**